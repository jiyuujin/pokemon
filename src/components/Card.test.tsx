import React from 'react'
import renderer from 'react-test-renderer'

import { Card } from './Card'

test('Render component', () => {
    const component = renderer.create(
        <Card
            pokemon={{
                name: 'unown',
                url: 'https://pokeapi.co/api/v2/pokemon/201/',
            }}
        />
    )
    expect(component).toMatchSnapshot()
})
