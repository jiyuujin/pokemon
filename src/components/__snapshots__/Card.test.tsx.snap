// Vitest Snapshot v1

exports[`Render component 1`] = `
<a
  href="https://www.pokemon.com/us/pokedex/unown"
  rel="noopener noreferrer"
  target="_blank"
>
  <img
    alt="unown image"
    className="gallery__item__img"
    src="https://img.pokemondb.net/artwork/large/unown.jpg"
  />
  <div
    className="gallery__item__caption"
  >
    Unown
  </div>
</a>
`;
